tests/schemas/newsfeed.schema.js

```diff
@@ -22,7 +22,8 @@ const newsfeedArticleSchema = {
 	source: Joi.string().pattern(/^\b[a-z]+(?:_[a-z]+){1,2}\b$/).required(),
 	source_id: Joi.string().required(),
 	ctime: Joi.number().integer().positive().required(),
-	mtime: Joi.number().integer().positive().required(),
+	mtime: Joi.number().integer().positive().allow(null)
+		.required(),
```

Comment:
Why do we need to allow `null` values?

Reply:
There is no need to allow `null`. It was a mistake and I reverted it back.